platform/zoul/remote-revb/board.c

```diff
@@ -39,13 +39,111 @@
 /*---------------------------------------------------------------------------*/
 #include "contiki-conf.h"
 #include "antenna-sw.h"
+#include "dev/ioc.h"
 #include <stdint.h>
 #include <string.h>
 /*---------------------------------------------------------------------------*/
+#define CLEAR(x, y, z) GPIO_SOFTWARE_CONTROL(GPIO_PORT_TO_BASE(x), GPIO_PIN_MASK(y)); \
+                       ioc_set_over(x, y, z);     \
+                       GPIO_SET_OUTPUT(GPIO_PORT_TO_BASE(x), GPIO_PIN_MASK(y));  \
+                       GPIO_CLR_PIN(GPIO_PORT_TO_BASE(x), GPIO_PIN_MASK(y));
+
+#define SET(x, y, z)   GPIO_SOFTWARE_CONTROL(GPIO_PORT_TO_BASE(x), GPIO_PIN_MASK(y)); \
+                       ioc_set_over(x, y, z);     \
+                       GPIO_SET_OUTPUT(GPIO_PORT_TO_BASE(x), GPIO_PIN_MASK(y));  \
+                       GPIO_SET_PIN(GPIO_PORT_TO_BASE(x), GPIO_PIN_MASK(y));
+
+#define INPUT(x, y, z) GPIO_SOFTWARE_CONTROL(GPIO_PORT_TO_BASE(x), GPIO_PIN_MASK(y)); \
```

Comment:
I mean that `z` should be used with `ioc_set_over()`, not that it should be removed.